src/main/java/frc/robot/Robot.java

```diff
@@ -34,13 +34,19 @@ public void robotInit() {
 
     robotContainer = new RobotContainer();
 
-    // TODO: change to correct paths
-    chooser.setDefaultOption("Taxi", "taxi");
-    chooser.addOption("One Ball", "one ball");
-    chooser.addOption("Two Ball", "two ball");
-    chooser.addOption("Three Ball", "three ball");
-    chooser.addOption("Four Ball", "four ball");
-    SmartDashboard.putData("Auto Mode", chooser);
+    chooser.setDefaultOption("Taxi", Constants.Auton.TAXI); // default is taxi mode
+
+    chooser.addOption("Taxi", Constants.Auton.TAXI);
+    chooser.addOption("One Ball", Constants.Auton.ONEBALL);
+    chooser.addOption("Two Ball", Constants.Auton.TWOBALL);
+    chooser.addOption("Three Ball", Constants.Auton.THREEBALL);
+    chooser.addOption("Four Ball", Constants.Auton.FOURBALL);
+    chooser.addOption("Five Ball", Constants.Auton.FIVEBALL);
+    chooser.addOption("Test Mode", Constants.Auton.TEST);
+
+    SmartDashboard.putData(
+        "Auto Mode",
+        chooser); //  TODO: find a way to put it into desired specific named tabs such as "Auton"
```

Comment:
https://github.com/caseyjbrotherton/ShuffleBoardExamples/blob/main/src/main/java/frc/robot/subsystems/ExampleSubsystem.java#L38

Has examples about tabs.  Would suggest talking with @Asher123456789 and @Majorferguson21 as they were looking at moving stuff to an "Operator" tab and a "Driver" tab for competition.